examples/cuda/examples/maxwell/std.cuh

```diff
@@ -49,9 +49,9 @@ void run_std(float dt, bool write_vtk, std::size_t n_inner_iterations,
 
   const std::size_t n_threads = std::thread::hardware_concurrency();
   std::vector<std::thread> threads(n_threads);
-  std::barrier barrier(n_threads);
```

Comment:
Why is this change needed? The purpose of this example is to illustrate naive C++ implementation. I think seeing `cuda` here is misleading even if it's the same thing.